ballerina/tests/asb_sender_receiver_negative_tests.bal

```diff
@@ -52,6 +55,46 @@ function testReceivePayloadWithIncorrectExpectedType() returns error? {
     groups: ["asb_sender_receiver_negative"],
     dependsOn: [testReceivePayloadWithIncorrectExpectedType]
 }
+function testInvalidComplete() returns error? {
+    log:printInfo("[[testInvalidComplete]");
```

Comment:
We do not need these kind of logs right ? If there is an error it will be automatically logged with the stacktrace which we can use to pinpoint the issue. So, I feel that these logs are bit redundant.

Reply:
Yes, we can consider using the debug log level for debugging purposes as suggested.